### PR TITLE
Include send2cgeo into search

### PR DESCRIPTION
**Pull request by Janbk**
I often download a complete search result or the 10 most faved caches in an area. With this modification a new column is added to the search results with a  send2cgeo image as link. Perhaps this feature is useful for a lot of people.

### CODE REVIEW

**Comment by Lineflyer** [comment]
Tested and working fine for me.

@mucek4 
Any problem in using that URL for getting the image to click on?: http://send2.cgeo.org/content/images/send2cgeo.png

**Comment by mucek4** [comment]
Yes.
a) Image is too big
b) no https

**Comment by Lineflyer** [comment]
Size on the page looks ok:
http://i.imgur.com/JQYK6Ww.png

File: send2cgeo/send2cgeo.user.js [hunk 1]
```diff
@@ -74,7 +75,30 @@ s.textContent =  '(' + function() {
              + '<span>Send to c:geo</span>';
 
     map.innerHTML = map.innerHTML.replace('Log Visit</span>', html);
-  } else if(document.getElementById('ctl00_ContentBody_CoordInfoLinkControl1_uxCoordInfoCode') != null){
+  } else if(document.getElementById('searchResultsTable') != null){
+    // geocaching.com seach
```
Comment (Bananeweizen) [inline send2cgeo/send2cgeo.user.js]: typo in "search"

**Comment by Bananeweizen** [comment]
I like the that change very much. @mucek4 I guess we can always use the http_s_ version of the image, independent of whether the geocaching.com page is browsed via http or https, right?

**Comment by Janbk** [comment]
Thank you for all the comments.
I update the branch including the remarks. The HTTPS is no problem because the actual sending is done by HTTPS anyway.

I do not like the size of the image, too. But I needed an image with 50x50px hosted on a trusted domain. Geocaching.com was no good source because the phone image used in other parts of the script is to small, so I was searching on cgeo.org. Do you have a smaller version of the logo hidden anywhere?

**Comment by mucek4** [comment]
Resize it and commit it?

**Comment by Janbk** [comment]
I just uploaded a solution for your suggestion @mucek4, but I do not like it. The URL has to include my username and the branch name the Github webpage created. So the current URL is

> https://raw.githubusercontent.com/Janbk/cgeo/patch-1/send2cgeo/send2cgeo.png

but after merging it could be

> https://raw.githubusercontent.com/cgeo/cgeo/master/send2cgeo/send2cgeo.png

This could work but it looks much better if the image is hosted by cgeo. This script works, so it is fine for me. :D

**Comment by kumy** [comment]
@Janbk I think @mucek4 talked about this repo https://github.com/cgeo/cgeo.github.io/tree/master/images
Images links would be like http://www.cgeo.org/images/cgeo-logo.png

**Comment by Janbk** [comment]
Yep, that is what I was looking for. Thank you @kumy for this ging.

**Comment by mucek4** [comment]
More or less no. It's http again and I am not sure if https works.

**Comment by mucek4** [comment]
Here you go https://send2.cgeo.org/send2cgeo.png

**Comment by kumy** [comment]
you're right, no "trusted" https on www.cgeo.org
For my information, which one is the good repo?

**Comment by Lineflyer** [comment]
@Janbk  Can you please update your PR with the new URL as given by @mucek4 ?

**Comment by Janbk** [comment]
Hi, I changed the last commit to remove the image from commit history and used a nice and secure url for the png file.

Thank you for your time and comments.

**Comment by Lineflyer** [comment]
Thanks for your contribution!
There is an extra space in the URL, will remove that after merging.

We will probably use that change to introduce a new release management for the script (as we also recently changed our website and this is still one open issue). So it might take some time until this update is officially published via the website.

**Pull request merged**